### DISCUSSION TITLE
Problems with entering COM / NAV frequencies on Streamdeck XL

### DISCUSSION THREAD

**Discussion by stenax** [nguyenquyhy/Flight-Tracker-StreamDeck#283] [Q&A]
Hello,
the frequency entries worked without problems on the "normal" Streamdeck 3x5. There were two pages on the Streamdeck 3x5.
- First page: long press on COM1, for example. 
- Second page: The page with the number block opened, on which the frequency could be entered.

Now I have switched to the **Streamdeck XL** and there are problems:

- The buttons (for example COM1) and the numeric keypad for entering the frequencies are only displayed on one Streamdeck-side. 
If I press (for example) COM1 for a long time, nothing happens and I cannot enter a frequency via the number pad.

I have reinstalled both the Streamdeck software and Flighttracker. Unfortunately without success.

This is what it looks like on the Streamdeck XL: [click](https://cdn.discordapp.com/attachments/802263624076099644/871130732796465152/unknown.png) (Discord)

Can anyone help me? Thank you!

**Comment by nguyenquyhy** [comment]
Did you rename that profile to FlightTracker? You must not rename or edit that profile unfortunately. I would suggest to remove that FlightTracker profile in the screenshot and reinstall the plugin so it will add back the original Numpad profile.

**Comment by stenax** [comment]
Thank you for your advice! Unfortunately no improvement.
I have done the following:
- I have deinstalled the Streamde software
- I deleted all Streamdeck data from %appdata%.
- I installed the Streamdeck software again
- I installed Flighttracker first

Flighttracker has created a profile with the name: "Profile 1 copy"
It looks like the screenshot above. A long press on the COM button has no function. A 2nd page (Numpad) was NOT installed. There is no button für entering the Squawk Code.

What can I do?

**Reply by nguyenquyhy** [reply]
That is strange. The plugin should create just a "Numpad" profile. It should not create any "Profile 1 copy". Can you reinstall again and make sure there is no existing profile named Numpad before the installation.

If that still fails, maybe you can try to rename the "Profile 1 Copy" to "Numpad".

**Reply by stenax** [reply]
I'm afraid it doesn't work.
I have uninstalled everything. Only the default profile is in the stream deck. Flightracker always installs a profile with the name "Profile 1 copy". Unfortunately, renaming it to Numpad doesn't work either.
Do you have a Discord? Then I could share my screen. Maybe you can help then?

**Reply by nguyenquyhy** [reply]
Yes you can join our Discord server here https://discord.gg/azqQrWWEbV. But I think I found the issue. We messed up the name sometimes before. Thanks for letting me know.

If you ping me (Hy) in that Discord server, I'll send you a new build to see if it fixes the issue.

**Reply by stenax** [reply]
Solved in Discord. Thank you very much!

**Comment by nicolasaujoux** [comment]
Hello and thank you for your awsome piece of software. I encountered the same issue, any chance that there is a fix on the way ?

**Reply by nguyenquyhy** [reply]
I am working on releasing a fix today in version 0.38.
If you can't wait for that, feel free to ping me (Hy) in Discord (link above) and I'll tell you the workaround. I'm quite surprised I didn't hear about this earlier since this bug has been there for quite a while.

**Reply by nicolasaujoux** [reply]
No problem, I can definitely wait a bit ! Thanks again :)

**Reply by nguyenquyhy** [reply]
Actually, I would appreciate if you can ping me in Discord. I can give you the new version in around an hour to test. I don't have an XL at the moment so I couldn't know exactly how the fix would affect users with the XL.
